apps/src/templates/rubrics/LearningGoals.jsx

```diff
@@ -211,7 +242,6 @@ export default function LearningGoals({
     INVALID_UNDERSTANDING
   );
   const [aiFeedback, setAiFeedback] = useState(-1);
-  const [aiEvidence, setAiEvidence] = useState([]);
```

Comment:
This isn't really state. It is something calculated from state. So it got converted to a memo.